opensrp-app/src/main/java/org/smartregister/sync/intent/DistrictFacilitiesIntentService.java

```diff
@@ -0,0 +1,38 @@
+package org.smartregister.sync.intent;
+
+import android.content.Intent;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import org.joda.time.DateTime;
+import org.smartregister.domain.LocationProperty;
+import org.smartregister.sync.helper.LocationServiceHelper;
+import org.smartregister.util.DateTimeTypeConverter;
+import org.smartregister.util.PropertiesConverter;
+
+import timber.log.Timber;
+
+public class DistrictFacilitiesIntentService extends BaseSyncIntentService {
+    private static final String TAG = "LocationIntentService";
```

Comment:
You  might consider using the same class name for the tag to follow the convention.